node/pkg/chain/helper/types.go

```diff
@@ -90,4 +96,9 @@ const (
 	SignerPk        = "SIGNER_PK"
 	EthProviderUrl  = "ETH_PROVIDER_URL"
 	EthReporterPk   = "ETH_REPORTER_PK"
+
+	SignerRenewInterval       = 12 * time.Hour
+	SignerRenewThreshold      = 7 * 24 * time.Hour
```

Comment:
minor suggestion, how about declaring these numbers as constants or env variables? It might be better for readability and in case if we want to update these values in the future

Reply:
sure, I can set it as default value only in case when env variable is not provided